container_pipeline/lib/command.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run_cmd(cmd, shell=False):
```

Comment:
I'm not able to comprehend having a dedicated file for just one function. Not that it's bad. But we also have `utils.py` which is having a number of utilities that we need to reuse across various files. Why haven't we included this function as a part of `utils.py`?